build/BuildDefaults.props

```diff
@@ -6,10 +6,10 @@
     <!-- https://github.com/dotnet/cli/issues/8800 -->
     <IncludeNuGetPackageArchive Condition=" $(Architecture.StartsWith('arm')) ">false</IncludeNuGetPackageArchive>
     <IncludeNuGetPackageArchive Condition=" '$(IncludeNuGetPackageArchive)' == '' ">true</IncludeNuGetPackageArchive>
-    <SkipBuildingInstallers Condition=" '$(SkipBuildingInstallers)' == '' AND ($(Rid.StartsWith('rhel.6')) OR $(Rid.StartsWith('alpine.3.6')))">true</SkipBuildingInstallers>
+    <SkipBuildingInstallers Condition=" '$(SkipBuildingInstallers)' == '' AND ($(Rid.StartsWith('rhel.6')) OR $(Rid.StartsWith('linux-musl')))">true</SkipBuildingInstallers>
     <SkipBuildingInstallers Condition=" '$(SkipBuildingInstallers)' == '' ">false</SkipBuildingInstallers>
-    <UsePortableLinuxSharedFramework Condition=" '$(UsePortableLinuxSharedFramework)' == '' AND '$(OSPlatform)' == 'linux' AND '$(Rid)' != 'rhel.6-x64' AND '$(Rid)' != 'alpine.3.6-x64' ">true</UsePortableLinuxSharedFramework>
-    <IncludeSharedFrameworksForBackwardsCompatibilityTests Condition=" $(IncludeSharedFrameworksForBackwardsCompatibilityTests) == '' AND '$(Rid)' != 'linux-x64' AND '$(Rid)' != 'rhel.6-x64' AND '$(Rid)' != 'alpine.3.6-x64'">true</IncludeSharedFrameworksForBackwardsCompatibilityTests>
+    <UsePortableLinuxSharedFramework Condition=" '$(UsePortableLinuxSharedFramework)' == '' AND '$(OSPlatform)' == 'linux' AND '$(Rid)' != 'rhel.6-x64' AND '$(Rid)' != 'linux-musl-x64' ">true</UsePortableLinuxSharedFramework>
```

Comment:
(no change needed. Just for discussion)

This change to `linux-musl` is muddying up the definition of `portable` in all our repos.  Technically, the `linux-musl` RID is "portable" and is "linux".  It just isn't the glibc based portable version.  Even in our core repos, we pass in `portable=false` when building for `linux-musl`, but it actually is portable across any musl based linux.

I'm not sure how to fix this, but just wanted to get this comment out there to get others thinking about it.

Reply:
I agree that this should be fixed at some point. Currently, at many places, portable=true means linux-x64 RID is used. This will need to change.

Reply:
Yeah I started by trying to use reuse portable but there are too many places that assume that means "linux" so I didn't have enough time to flush that out. If we care enough we can fix that in the future but ultimately I don't think it matters too much.